core/variant/callable.cpp

```diff
@@ -327,34 +328,47 @@ Callable::operator String() const {
 	}
 }
 
+Callable Callable::create(const Variant &p_variant, const StringName &p_method) {
+	ERR_FAIL_COND_V_MSG(p_method == StringName(), Callable(), "Method argument to Callable::create method must be a non-empty string.");
+
+	switch (p_variant.get_type()) {
+		case Variant::NIL:
+			return Callable(ObjectID(), p_method);
+		case Variant::OBJECT:
+			return Callable(p_variant.operator ObjectID(), p_method);
+		default:
+			return Callable(memnew(VariantCallable(p_variant, p_method)));
+	}
+}
```

Comment:
It'd be nice to have some unit tests (either in core, GDScript, or both) verifying that all these cases work as intended.

Reply:
I added a GDScript test, but skipped the core for now since there is no `tests/core/variant/callable.h` file.